learning-another-language/variables.md

```diff
@@ -68,6 +68,41 @@ var jingsFavColor = 'green';
 ```
 <!-- prettier-ignore-end -->
 
+### Scope of var vs let & const
+
+When creating a variable using `let` and `const` will scope the variable to the block it is in.  In the below example `i` and `name` are defined inside of the `for-of` loop and are scoped to the loop so are not accessable outside of the loop.  This applies to all blocks of code in JavaScript, except conditionals (if statements).  
```

Comment:
The true block in the example is the stuff in the {}. It feels potentially confusing to talk about the variable definition in the head while talking about the block. So I would introduce the main block stuff with very strong statements about when the variable is visible, and then as a by the way, note that for for loops, the condition head is considered part of the block. I don't feel like the notice about ifs is needed, since it's syntactically illegal to define a variable in an if condition (wants an expression, but var/let/const are statements). This also rules out while loops (expression required), so I feel it's better to avoid mentioning it.

And single spaces after periods, please (consistency).

```suggestion
When creating a variable, using `let` and `const` will scope the variable to the block that contains it. In the example below, `name` is defined inside of the `for-of` loop's block. It is _scoped_ to the block. It is not accessible outside of the block. This applies to all blocks of code in JavaScript.

In JavaScript, the `for` family of loops (`for`, `for-in`, `for-of`) consider the loop's condition to be within the block scope of the loop as well. So the variable `i`, defined in the condition of the `for-of` loop, is also _scoped_ to the block, and cannot be accessed outside of the block.
```